tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -354,13 +354,13 @@ function populate_perf_filesystems
 
 function get_nfilesystems
 {
-	typeset filesystems=( $TESTFS )
+	typeset filesystems=("$TESTFS")
```

Comment:
This seems to be an array initialization. Do we really want to convert empty variable into one element array of empty string?

Reply:
You're right, this should not have quotes.